src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/media-edit-overlay/cropping-slide.js

```diff
@@ -0,0 +1,516 @@
+/*
+ * This file is part of the Sulu CMF.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+/**
+ * @class CroppingSlide
+ * Class which handles the cropping-slide in the media-edit overlay.
+ * Note that this is not an aura-component but a simple require-js dependency.
```

Comment:
The media-overlay and the cropping-slide conceptually more tightly coupled. They exchange different data and configurations. For example through `getSlideDefinition` the overlay component can retrieve the slide configuration-object from the cropping-slide. I did not want to write the behavior of the slide into the media-overlay component like for the first slide. Therefore I created a separate require module, which encapsulates the logic and configuration.